feature/search/src/main/kotlin/uk/govuk/app/search/ui/SearchScreen.kt

```diff
@@ -60,6 +62,11 @@ internal fun SearchRoute(
         onClear = {
             viewModel.onClear()
         },
+        onClick = { title, url ->
+            viewModel.viewModelScope.launch {
```

Comment:
The view model scope should be launched within the view model function.

Reply:
Fixed up